.travis.yml

```diff
@@ -4,4 +4,12 @@ services:
     - docker
   
 script:
-    - ./test-build.sh
+    - ./test-build.sh $NODE_VERSION
+
+env:
+    matrix :
+        - NODE_VERSION: '0.10'
```

Comment:
v5 is EOL and no longer maintained. When this PR started the EOL for v0.10 was way out in the future and we still have not removed it from the repo yet. There will most likely be a follow up PR to remove it in a few weeks.